<?php

namespace App\Exceptions;

use Exception;

class UserHasBeenTakenException extends Exception
{
        
    /**
     * Converte a exceÃ§Ã£o em uma resposta HTTP
     */
    public function render()
    {
        return response()->json([
            'error'   => class_basename($this), // nome da classe -> padrÃ£o para o front
            'message' => 'User has been taken.',
        ], 400);
    }
}
